packages/common/src/helpers/errors.ts

```diff
@@ -23,6 +23,8 @@ export function getMessage(error: unknown): string {
     return 'Error is not specified.';
   }
 
+  console.error(error);
```

Comment:
Isn't https://github.com/eclipse-che/che-dashboard/pull/382/files#diff-49e7e170b9486c6ee4b7e0258526f1cf45708b29d55e5d791eba4d4cc63d97beR63 enough?

Reply:
It wasn't enough, because the origin error could include additional information. It will be useful to see.